content/pt-br/docs/concepts/overview/working-with-objects/namespaces.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Namespaces
+content_type: concept
+weight: 30
+---
+
+<!-- overview -->
+
+Em Kubernetes, _namespaces_ disponibilizam um mecanismo para isolar grupos de recursos dentro de um único cluster. Nomes de recursos precisam ser únicos dentro de um namespace, porém podem se repetir em diferentes namespaces. Escopos baseados em namespaces são aplicáveis apenas para objetos com namespace _(como: Deployments, Services, etc)_ e não em objetos que abrangem todo o cluster _(como: StorageClass, Nodes, PersistentVolumes, etc)_.
+
+<!-- body -->
+
+## Quando Utilizar Múltiplos Namespaces
+
+Namespaces devem ser utilizados em ambientes com múltiplos usuários espalhados por diversos times ou projetos. Para clusters com poucos ou até algumas dezenas de usuários, você não deveria precisar criar ou pensar a respeito de namespaces. Comece a utilizar namespaces quando você precisar das funcionalidades que eles oferecem.
+
+Namespaces oferecem escopo para nomes. Nomes de recursos precisam ser únicos dentro de um namespace, porém não em diferentes namespaces. Namespaces não podem ser aninhados dentro de outros namespaces e cada recurso Kubernetes pode pertencer à apenas um namespace.
+
+Namespaces nos permitem dividir os recursos do cluster entre diferentes usuários (via [resource quota](/docs/concepts/policy/resource-quotas/)).
+
+Não é necessário utilizar múltiplos namespaces para separar recursos levemente diferentes, como diferentes versões de um mesmo software: use {{< glossary_tooltip text="labels" term_id="label" >}} para distinguir recursos dentro de um mesmo namespace.
+
+## Trabalhando com Namespaces
+
+Criação e eliminação de namespaces estão descritas na 
+[documentação de namespaces do guia de administradores](/docs/tasks/administer-cluster/namespaces).
+
+{{< note >}}
+    Evite criar namespaces com o prefixo `kube-`, já que este prefixo é reservado para namespaces do sistema Kubernetes.
+{{< /note >}}
+
+### Visualizando namespaces
+
+Você pode obter uma lista dos namespaces atuais dentro de um cluster com:
+
+```shell
+kubectl get namespace
+```
+```
+NAME              STATUS   AGE
+default           Active   1d
+kube-node-lease   Active   1d
+kube-public       Active   1d
+kube-system       Active   1d
+```
+
+Kubernetes se inicializa com quatro namespaces iniciais:
```

Comment:
This sentence can be improved, something like: `O Kubernetes é inicializado com quatro namespaces: ` or something like that, what do you think?